federation/pkg/federation-controller/configmap/configmap_controller.go

```diff
@@ -330,7 +330,10 @@ func (configmapcontroller *ConfigMapController) reconcileConfigMap(configmap typ
 		glog.Errorf("Error in retrieving obj from store: %v, %v", ok, err)
 		return
 	}
-
+	if util.IsFederationOnlyObject(configMap) {
```

Comment:
Why do we need this specific check?
There will be a check to determine which subset of clusters does the resource needs to be propagated to. That check should be enough to determine that a resource should not be propagated to any cluster?